wandb/sdk/wandb_run.py

```diff
@@ -1885,21 +1916,10 @@ def wrap_fallback() -> None:
     def _restore(self) -> None:
         logger.info("restore")
         # TODO(jhr): drain and shutdown all threads
```

Comment:
should we remove this todo?

Reply:
not for the new code.. there are no new threads in the user process.
for the old wrap_emu... maybe, but i was avoiding changing behavior there.